src/family/family.service.spec.ts

```diff
@@ -57,4 +60,33 @@ describe('FamilyService', () => {
       expect(family).toEqual(familyStub);
     });
   });
+
+  describe('create', () => {
+    it('should be defined', () => {
+      // eslint-disable-next-line @typescript-eslint/unbound-method
+      expect(service.create).toBeDefined();
+    });
+
+    it('should called with correct argument', async () => {
+      const data: CreateFamilyInput = { name: faker.name.fullName() };
+      const method = jest
+        .spyOn(prisma.family, 'create')
+        .mockResolvedValue(familyStub);
+      await service.create(data);
+
+      expect(method).toHaveBeenCalledWith({
+        data: { ...data, slug: slug(data.name) },
+      });
+    });
```

Comment:
this test case belongs to family service, not repo!
As I understood you write a test case to ensure `create` method on the family repo is called with the correct args or not,
you should test `create` method on the service called with appropriate args or not

Reply:
I don't understand your point here.

Reply:
you should write test for `create` method on service is called with expected args, not create method on `prisma.family.create`

Reply:
What's the point to check which parameters that I passed in service.create method? (I will call it directly in test case and there is no way to be wrong)

I should check is service.create method calls prisma service with correct parameters or not. (I called service.create on line 75)

Reply:
Actually, both of them are useless :)
no difference between them,
in the context of unit test, we try to ensure is our unit work correctly and whether We expect it or not,
based on this definition, checking if the Prisma family `create` method called with appropriate args is not belong to this definition

Reply:
Yes, you're right.

What's your suggestion?